packages/smartcontracts/src/tests/BridgeProxy.flushFund.spec.ts

```diff
@@ -0,0 +1,143 @@
+  });
+
+  it('Should not flush fund for a token if it is in the adding period', async () => {
+    const { proxyBridge, flushReceiveSigner, testToken } = await loadFixture(deployContracts);
+    // latestResetTimestamp = block.timestamp + 59 = X + 59
+    await proxyBridge.addSupportedTokens(testToken.address, toWei('10'), (await time.latest()) + 60);
+    // block.timestamp = X + 1
+    await testToken.mint(proxyBridge.address, toWei('100'));
+    const balanceBeforeFlush = await testToken.balanceOf(flushReceiveSigner.address);
+    // block.timestamp = X + 2
+    await proxyBridge.flushFund();
+    const balanceAfterFlush = await testToken.balanceOf(flushReceiveSigner.address);
+    expect(balanceAfterFlush).to.equal(balanceBeforeFlush);
+  });
+
+  it('Should not flush fund for a token if it is in the change allowance period', async () => {
+    const { proxyBridge, flushReceiveSigner, testToken, defaultAdminSigner } = await loadFixture(deployContracts);
+    // latestResetTimestamp = block.timestamp + 59 = X + 59
+    await proxyBridge.addSupportedTokens(testToken.address, toWei('10'), (await time.latest()) + 60);
+    const latestResetTimestampAfterAddingSupport = (await proxyBridge.tokenAllowances(testToken.address))
+      .latestResetTimestamp;
+    expect(latestResetTimestampAfterAddingSupport).to.equal((await time.latest()) + 59);
+    // block.timestamp = X + 1
+    await testToken.mint(proxyBridge.address, toWei('100'));
+    // create a new block with timestamp = X + 1 + 60 = X + 61 > X + 59 = latestResetTimestamp
+    await time.increase(60);
+    await testToken.approve(proxyBridge.address, ethers.constants.MaxUint256);
+    await testToken.mint(defaultAdminSigner.address, toWei('5'));
+    await proxyBridge.bridgeToDeFiChain(ethers.constants.AddressZero, testToken.address, toWei('5'));
+    expect((await proxyBridge.tokenAllowances(testToken.address)).latestResetTimestamp).to.equal(
+      latestResetTimestampAfterAddingSupport,
+    );
+    // new latestResetTimestamp = block.timestamp + 1 days = Y + 1 days
+    await proxyBridge.changeDailyAllowance(testToken.address, toWei('20'), (await time.latest()) + ONE_DAY + 1);
+    expect((await proxyBridge.tokenAllowances(testToken.address)).latestResetTimestamp).to.equal(
+      (await time.latest()) + ONE_DAY,
+    );
+    const balanceBefore1stFlush = await testToken.balanceOf(flushReceiveSigner.address);
+    // block.timestamp = Y + 1 < Y + 1 days = new latestResetTimestamp
+    await proxyBridge.flushFund();
+    const balanceAfter1stFlush = await testToken.balanceOf(flushReceiveSigner.address);
+    expect(balanceAfter1stFlush).to.equal(balanceBefore1stFlush);
+    // new block with block.timestamp = Y + 1 + ONE_DAY
+    await time.increase(ONE_DAY);
+    const balanceBefore2ndFlush = await testToken.balanceOf(flushReceiveSigner.address);
+    const balanceBridgeBefore2ndFlush = await testToken.balanceOf(proxyBridge.address);
```

Comment:
```suggestion
    const bridgeBalanceBefore2ndFlush = await testToken.balanceOf(proxyBridge.address);
```

Same for the other variables as well.